src/mi/Repl.cs

```diff
@@ -214,6 +214,7 @@ private void HandleEscape(ObservableCollection<string> document, SubmissionView
             document.Clear();
             document.Add(string.Empty);
             view.CurrentLine = 0;
+            view.CurrentCharacter = 0;
```

Comment:
Why is this needed?

Reply:
This proposed change is to fix an ArgumentOutOfRangeException. The setter of [CurrentLine](https://github.com/terrajobst/minsk/blob/7abf535b0d1dd825718584c713372b97196ed215/src/mi/Repl.cs#L107) checks if the assigned new line is different from the current value.
When handling escape for single line input (or for multi-line input but the cursor is already at the first line), the current line is already 0, and thus the character isn't moved to 0. That leaves the character in the same place as it was before handling escape. If at that point you try to use backspace to clear the white spaces, an exception is thrown, because the caret is in an invalid position.